devel/shellcheck/Portfile

```diff
@@ -30,7 +30,7 @@ if { ${os.platform} eq "darwin" && ${os.major} >= 20 } {
     homepage        https://github.com/koalaman/shellcheck
     livecheck.type  regex
     livecheck.url   https://github.com/koalaman/shellcheck/tags
-    livecheck.regex "archive/v(\[^\"]+)\\.tar\\.gz"
+    livecheck.regex "archive/refs/tags/v(\[^\"]+)\\.tar\\.gz"
```

Comment:
Does this port need to specify the regex at all, the above looks like the default should work ok ?

Reply:
Please try removing all the custom livecheck settings and see if it works. If not, then setting them via the github PG would be better.

Reply:
I have refactored the port to use the PG including for livecheck and share the `github.setup` line.